share/completions/mkdir.fish

```diff
@@ -0,0 +1,9 @@
+complete -c mkdir -s m -l mode --description 'Set file mode (as in chmod)' -x
+complete -c mkdir -s p -l parents --description 'Make parent directories as needed'
+complete -c mkdir -s v -l verbose --description 'Print a message for each created directory'
+complete -c mkdir -s Z --description 'Set SELinux security context of each created directory to the default type'
```

Comment:
Yeah, testing for a feature directly is usually the way to go. 

I think perhaps further you should also test, somehow, for SELinux on Linux, and if that doesn't exist, don't show the --context=CTX (-Z CTX) options.